core/services/headtracker/head_tracker.go

```diff
@@ -242,7 +251,7 @@ func (ht *HeadTracker) backfiller() {
 					panic(fmt.Sprintf("expected `models.Head`, got %T", head))
 				}
 				{
-					ctx, cancel := utils.ContextFromChan(ht.chStop)
```

Comment:
using ht.chStop here is not needed, as it's already used in the surrounding select